src/main/jte/calendar.jte

```diff
@@ -40,6 +53,8 @@
                 events: '/api/events'
             });
 
+            // set locale to german
+            calendar.setOption('locale', 'de');
```

Comment:
We support english users too

Reply:
fair i guess but switching between german and englisch is something for the implementing phase as we would also have to change the time format from 24h to 12h format and dd-MM to MM-dd. And our main formating style until now was mostly 24h and dd-MM so i would say that we keep it in german format for now. 

Do you agree?